docs/evaluation/load_test.Rmd

```diff
@@ -0,0 +1,756 @@
+}
+
+```
+
+
+##### Test execution
+
+The test execution requires two functions, the first one `create_dockerfile` and `run_test`. The first one creates a dockerfile with the configuration to build the environment of the test and the second one executes the test. The `create_dockerfile` function runs inside the `run_test` as it takes the parameters of the test to generate and specific dockerfile environment.
+
+###### Dockerfile
+
+This function creates a dockerfile with the required parameters to containers either for a **creation session** or a **examination session**. This includes authentication parametes, configuration files and folders with required files (workpaces). This dockerfile copies into the envieronment the `test_o2r.r` and the `o2r_test/o2r_sessions.R` scripts. Both files are requires to run the containers.
+
+```{r dockerfile}
+
+library(dockerfiler)
+
+# create_dockerfile
+### Parameters
+# cookie -> o2r`connec.sid` cookie
+# ednpoint ->  o2r endpoint
+# special_issue_folder -> path to folder with ERC/Workspaces for special issue
+# special_issue_conf_file ->path to configuration file for Creation sessions (special issue creation)
+# examination_conf_file -> path to configuration file for Examination sessions (Special issue User interaction) 
+# creation_folder -> path to folder with ERC/Workspaces for "late submissions" (Special issue User interaction) 
+# creation_conf_file -> path to configuration file for Creation sessions (Special issue User interaction / "late submission") 
+
+create_dockerfile<-function(cookie,
+                            endpoint,
+                            special_issue_folder,
+                            special_issue_conf_file,
+                            examination_conf_file,
+                            creation_folder,
+                            creation_conf_file){
+  mydock<-Dockerfile$new("rocker/tidyverse:3.6.3")
+  mydock$ENV("COOKIE", cookie)
+  mydock$ENV("ENDPOINT", endpoint)
+  mydock$RUN("mkdir /usr/scripts")
+  mydock$WORKDIR("/usr/scripts")
+  mydock$COPY("test_o2r.r","test_o2r.r")
+  mydock$COPY("o2r_test","o2r_test")
+  mydock$COPY(special_issue_folder,special_issue_folder)
+  mydock$COPY(special_issue_conf_file,special_issue_conf_file)
+  mydock$COPY(examination_conf_file,examination_conf_file)
+  mydock$COPY(creation_folder,creation_folder)
+  mydock$COPY(creation_conf_file,creation_conf_file)
+  mydock$write()
+}
+
+```
+
+###### run_test
+
+This functions run the test based on the configuration parameters. During the test the function gives an update of the stage (Special Issue creation or user interaction) and the corresponding creation or examination sessions. As an 'output' the test returns a folder with the results of every stage of the test. 
+
+```{r run_test}
```

Comment:
This code chunk has one really long function - please split the function up into multiple functions so the code becomes readable.